src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -624,10 +635,10 @@ Environment variable: `$WEB_EXT_TIMEOUT`
 
 #### `--id`
 
-A custom identifier string for the extension. This has no effect if the extension already declares an identifier in its manifest. This option may be useful for signing versions of an existing extension that you own. 
+A custom identifier string for the extension. This has no effect if the extension has declared an identifier in its manifest. This option may be useful for signing versions of an extension you own. 
 
 ::: note alert
-This option cannot be used when `--use-submission-api` is also used: the add-on ID must be specified in the `manifest.json` file.
+This option cannot be used when `--use-submission-api` is used: the add-on ID must be specified in the `manifest.json` file.
```

Comment:
@dotproto this would seem to suggest that `--id` has no purpose in version 8, correct?

Reply:
I believe you're correct. After reading through source and double checking with some local tests, there doesn't appear to be a way to use the `--id` flag in 8.0.0. 

I'll open a PR to remove it and update the `--help` docs.

Reply:
I've opened mozilla/web-ext#3124.